stable/rabbitmq-ha/templates/servicemonitor.yaml

```diff
@@ -14,8 +14,11 @@ spec:
       app: {{ template "rabbitmq-ha.name" . }}
       release: {{ .Release.Name }}
   endpoints:
-  - port: exporter
+  - port: {{ template "rabbitmq-ha.monitoringPortName" . }}
     interval: {{ .Values.prometheus.operator.serviceMonitor.interval }}
+    {{- if .Values.rabbitmqPrometheusPlugin.enabled }}
```

Comment:
If you're going to have an if statement here regardless, might as well move logic in `template "rabbitmq-ha.monitoringPortName"` here too.

Reply:
Alright. This was because if we give the possibility of changing native metrics port and endpoint, ServiceMonitor should reflect those changes in order to work.